### ISSUE TITLE
Bug: kusion preview generates an unnecessary ci-test folder

### ISSUE THREAD

**Issue by SparkYuan** [KusionStack/kusion#581]
### What happened?

kusion preview generates an unnecessary ci-test fold

### What did you expect to happen?

do not generate an unnecessary ci-test fold

### How can we reproduce it (as minimally and precisely as possible)?
```shell
>>> cat kcl.yaml
kcl_cli_configs:
  file:
    - microservice.k
  package_maps:
    k8s: ../../vendor/k8s_1.24

>>> cat stack.yaml
name: demo

>>> ls
total 12K
-rw-r--r-- 1 prahalad staff 93 Oct 20 16:56 kcl.yaml
-rw-r--r-- 1 prahalad staff 49 Oct 20 17:43 microservice.k
-rw-r--r-- 1 prahalad staff 22 Oct 20 16:56 stack.yaml

>>> kusion preview
 ✔︎  Generating Spec in the Stack api-data-sevbot...
Stack: api-data-sevbot  ID                                                      Action
* ├─                   policy/v1:PodDisruptionBudget:default:nginx-deployment  Create
* └─                   apps/v1:Deployment:default:nginx-deployment             Create

>>> ls
total 12K
drwxr-x--- 2 prahalad staff 64 Oct 20 22:51 ci-test

>>> kusion compile
 ✔︎  Generating Spec in the Stack api-data-sevbot...

>>> ls ci-test
total 4.0K
-rw-r--r-- 1 prahalad staff 832 Oct 20 22:51 stdout.golden.yaml
```

### Kusion version

<details>

```console
$ kusion version
# paste output here
```
releaseVersion: v0.9.0
gitInfo:
    latestTag: v0.9.0
    commit: 230d4cc37181e596942ce81a71452f84c47edfe0
    treeState: clean
buildInfo:
    goVersion: go1.19.13
    GOOS: linux
    GOARCH: amd64
    numCPU: 2
    compiler: gc
    buildTime: "2023-09-28 02:32:41"
dependency:
    kclGoVersion: v0.6.0-alpha.1
    kclPluginVersion: v0.5.0
</details>

### OS version

<details>

```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here

# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```

</details>

### Install tools

<details>

</details>

**Comment by healthjyk** [comment]
I didn't reproduce the bug, maybe your kusion version is not the latest(0.9.0).
Here is my result:
```
>>> cd single-stack-sample; tree .
.
├── README.md
├── dev
│   ├── kcl.mod
│   ├── kcl.mod.lock
│   ├── main.k
│   └── stack.yaml
└── project.yaml

1 directory, 6 files

>>> cd dev; kusion preview
 ✔︎  Generating Spec in the Stack dev...
Stack: dev  ID                                                  Action
* ├─     v1:Namespace:helloworld                             Create
* ├─     v1:Service:helloworld:helloworld-dev-nginx-private  Create
* └─     apps/v1:Deployment:helloworld:helloworld-dev-nginx  Create

>>> cd ..; tree .
.
├── README.md
├── dev
│   ├── kcl.mod
│   ├── kcl.mod.lock
│   ├── kusion_state.json
│   ├── main.k
│   └── stack.yaml
└── project.yaml

1 directory, 7 files
```

**Comment by ffforest** [comment]
New users should use the kpm route when using `kusion preview`.

**Comment by ffforest** [comment]
Resolved as part of #608
